tests/store/crud/add/test_store_add_base.py

```diff
@@ -153,3 +157,40 @@ def test_add_illumina_flow_cell_already_exists(
     # THEN a EntryAlreadyExistsError should be raised
     with pytest.raises(EntryAlreadyExistsError):
         store.add_illumina_flow_cell(illumina_flow_cell_dto)
+
+
+def test_create_pacbio_sequencing_run(store: Store):
+    # GIVEN a Store without Pacbio sequencing runs
+    assert not store._get_query(table=PacbioSequencingRun).all()
+
+    # GIVEN a PacBioSequencingRunDTO
+    pacbio_sequencing_run_dto = PacBioSequencingRunDTO(
+        instrument_name=RevioNames.WILMA, run_name="run_name"
+    )
+
+    # WHEN creating a Pacbio sequencing run in Store
+    store.create_pacbio_sequencing_run(pacbio_sequencing_run_dto)
+
+    # THEN the PacbioSequencingRun exists and has the correct attributes
+    pacbio_sequencing_run: PacbioSequencingRun = store._get_query(table=PacbioSequencingRun).one()
+    assert pacbio_sequencing_run.instrument_name == RevioNames.WILMA
+    assert pacbio_sequencing_run.run_name == "run_name"
+
+
+def test_create_pacbio_sequencing_run_already_exists(store: Store):
+    # GIVEN a Store without Pacbio sequencing runs
+    assert not store._get_query(table=PacbioSequencingRun).all()
+
+    # GIVEN a PacBioSequencingRunDTO
+    pacbio_sequencing_run_dto = PacBioSequencingRunDTO(
+        instrument_name=RevioNames.WILMA, run_name="run_name"
+    )
+
+    # GIVEN a Pacbio sequencing run in Store
+    store.create_pacbio_sequencing_run(pacbio_sequencing_run_dto)
+    store.commit_to_store()
+
```

Comment:
Reduce cognitive load
```suggestion
    # GIVEN a Store with a Pacbio sequencing run
    store.create_pacbio_sequencing_run(pacbio_sequencing_run_dto)
    store.commit_to_store()
    assert store._get_query(table=PacbioSequencingRun).one()

    # GIVEN a PacBioSequencingRunDTO
    pacbio_sequencing_run_dto = PacBioSequencingRunDTO(
        instrument_name=RevioNames.WILMA, run_name="run_name"
    )

```